sass/components/VmDetails/vm-details.scss

```diff
@@ -39,6 +39,11 @@
   width: 400px;
   flex: 1;
   flex-basis: auto;
+  margin-right: 8px;
+  
+  & > * {
```

Comment:
plus we also need this here

```
/* stylelint-disable plugin/selector-bem-pattern */

/* stylelint-enable */
```

because the tests are crashing